ui/imports/shared/popups/SendModal.qml

```diff
@@ -83,6 +87,71 @@ StatusDialog {
         property string totalTimeEstimate
         property double totalFeesInFiat
         property double totalAmountToReceive
+
+        property var selectedHolding: null
+        property var selectedHoldingType: HoldingTypes.Type.Unknown
+        readonly property bool isSelectedHoldingValidAsset: !!selectedHolding && selectedHoldingType === HoldingTypes.Type.Asset
+        property var hoveredHolding: null
+        property var hoveredHoldingType: HoldingTypes.Type.Unknown
+        readonly property bool isHoveredHoldingValidAsset: !!hoveredHolding && hoveredHoldingType === HoldingTypes.Type.Asset
+
+        function getHolding(holdingId, holdingType) {
```

Comment:
Can all of this be moved to the store?

Reply:
moved most logic to the store

Reply:
wondering if most of this is still needed here? can we not expose 2 properties from the HoldingSelector 1. SelectedToken and 2, The type? let me know what you think?

Reply:
The reason I'm doing this is because we use different models inside the selector (NestedCollectibles, a simpler model with less properties where a given row could represent either a collection or a collectible) and here we want the full Holding from the plain/flat Collectibles model. 
The only thing coming out of the HoldingSelector is the ID, which is used by the SendModal to obtain the full item from the other model.

Reply:
moved remaining functions to the store